src/engine/renderer/tr_fbo.cpp

```diff
@@ -507,6 +507,7 @@ void R_InitFBOs()
 		}
 	}
 
+	if ( !r_noportals->integer || r_liquidMapping->integer )
```

Comment:
r_noportals is not latched

Reply:
It's also CHEAT, although that doesn't make any sense

Reply:
I removed that.